README.md

```diff
@@ -44,7 +45,7 @@ HOW TO CONTRIBUTE
 -----------------
 
 The best way to contribute code is to fork this project, please read the [GUIDELINES.md](GUIDELINES.md) for further information.
```

Comment:
This EOL is not necessary and not rendered, so you can merge the two lines, it's a typo mistake I've done. :wink: